scripts/DDL.sql

```diff
@@ -2,7 +2,8 @@ CREATE SCHEMA jobs;
 
 CREATE TABLE jobs.users(
 	upn text PRIMARY KEY,
-	display_name text
+	display_name text,
+	last_entrance timestamp
```

Comment:
Why do we care about `last_entrance`? 🤔

Reply:
it's in order to always return the upserted user.  
After that i asked Chaki and Yuval and they wanted to make future use of it 👍